### DISCUSSION TITLE
Come Leonidas, let's reason together

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#2917] [Questions & Answers]
Sorry for the misleading title but since this is a discussion of an idea allow me the citation of 300.

As some of you know I am primarily a web designer and use mostly html and css.

OM provides the possibility of creating static blocks and widgets and positioning them in predetermined points of the theme or inserting static blocks in cms pages.

Web designers using this technique can sometimes create a lot of static blocks to insert into pages or other blocks.

the management of numerous blocks can become really dispersive and the search filters made available in the static blocks section become insufficient.

I was wondering if using the existing OM libraries it was possible to add and create a user interface similar to category management, namely:

have on the left the block tree example:

-block page cms home page
     |__top banner block
     |__block 2 banners
     |                 |__left banner
     |                 |__right banner
     |__html block
     |__Home Page custom css block

and keep the classic view on the right with the filters made available by OM.

sorry for my english i use google translate

**Comment by pauldpauld** [comment]
Is this any help?
https://github.com/Riki4321/magento-clever-cms

**Comment by Danieleeffe1** [comment]
Wonderful!
Tank's



Il mar 10 gen 2023, 23:36 pauldpauld ***@***.***> ha scritto:
